File: spring-camel-integration/src/main/java/org/apache/camel/learn/domain/Persona.java
package org.apache.camel.learn.domain;

import lombok.AccessLevel;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.FieldDefaults;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class Persona {

  String direccion;
  String nombres;
  String identificacion;
  int codigo;
  int pago;

  @Override
  public String toString() {
    return String.format("Nombre: %s, Direccion: %s, Identificacion: %s, Codigo: %s", nombres,
        direccion, identificacion, codigo);
  }
}

File: netmicro/Persona.cs
namespace netmicro;

public class Persona {

  public Persona(int codigo, string nombres, string direccion, string identificacion, int pago){
    this.pago = pago;
    this.codigo = codigo;
    this.nombres = nombres;
    this.direccion = direccion;
    this.identificacion = identificacion;
  }

  
  public int pago {get;set;}
  public int codigo {get;set;}
  public String nombres {get;set;}
  public String direccion {get;set;}
  public String identificacion {get;set;}
}